src/pool.c

```diff
@@ -1557,13 +1562,13 @@ hsk_peer_handle_proof(hsk_peer_t *peer, const hsk_proof_msg_t *msg) {
 
   hsk_map_del(&peer->names, msg->key);
 
-  hsk_name_req_t *req, *next;
+  hsk_tld_req_t *req, *next;
 
   for (req = reqs; req; req = next) {
     next = req->next;
 
     req->callback(
-      req->name,
+      (uint8_t *)req->tld,
```

Comment:
here as well.

Reply:
fixed in f5200ab1251950b897e8ef3f71f5a11737aeaa5a